stacks_queues/stack.py

```diff
@@ -21,18 +27,42 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store.head:
+            raise StackEmptyException
+            
+        value = self.store.tail.value
+        if self.store.head == self.store.tail:
+            self.store.head = self.store.tail = None
+        else:
+            self.store.tail.next = None
+            self.store.tail = self.store.tail.previous
+        
+        return value
+            
+            
+        
+
 
     def empty(self):
```

Comment:
✨ Again, this will work, however you could take advantage of `LinkedList`'s `empty` method